src/core/DynaRec_x64/gte_x64.cc

```diff
@@ -196,6 +196,29 @@ static uint32_t MFC2Wrapper(int reg) { return PCSX::g_emulator->m_gte->MFC2(reg)
 
 void DynaRecCPU::recMFC2() {
     if (_Rt_) {
+        const auto loadDelayDependency = getLoadDelayDependencyType(_Rt_);
+        if (loadDelayDependency != LoadDelayDependencyType::NoDependency) {
+            if (_Rd_ != 17) {
+                PCSX::g_system->message("Implement mean GTE behaviour for regs other than SZ1\n");
```

Comment:
Maybe add the register number / name in the message?

Reply:
This is debug logging that's not going to be in the final PR when it's past the draft stage